doc/source/user_documentation.rst

```diff
@@ -535,6 +531,105 @@ Here is an example of connectivity rules:
+
+	If ``synsPerConn`` == 1, and a list of sections is specified, synapses (one per presynaptic cell) will be placed in sections randomly selected from the list. If ``loc`` is also a list, locations will be picked randomly (note that the random section and location will go hand in hand, i.e., the same random index is used for both).
+
+* Setting ``synMech`` as a list
+	If ``synMech`` is a list, then for each mechanism in the list, a synapse will be created. The weights, delays, and locs can be specified as lists of the same length as ``synMech``, or a single value can be used for all.
+
+	.. code-block:: python
+
+		netParams.connParams['PYR->PYR'] = {
+			'preConds': {'pop': 'src'}, 'postConds': {'pop': 'dst'},
+			'synMech': ['AMPA', 'GABA'],
+			'weight': [0.1, 0.2], # individual weights for each synapse
+			'delay': 0.1, # single delay for all synapses
+		}
+		# This will create 2 synapses: one with AMPA synMech and weight 0.1
+		# and another with GABA synMech and weight 0.2. Both will have the same delay of 0.1.
+
+
+	However, for the sections, no such one-to-one correspondence to ``synMech`` elements applies. Instead, contents of ``secs`` are repeated for each synapse.
+
+	.. code-block:: python
+
+		netParams.connParams['PYR->PYR'] = {
+			'preConds': {'pop': 'src'}, 'postConds': {'pop': 'dst'},
+			'synMech': ['AMPA', 'GABA'],
+			'sec': ['soma', 'dend']
+		}
+		# Both AMPA and GABA synapses will span 'soma' and 'dend' sections
+
+	If you want to have different sections, separate connectivity rules (``connParams`` entries) should be defined for each synaptic mechanism.
+
+* Both ``synMech`` as a list and ``synsPerConn`` > 1
+	If you have both a N-element list of ``synMechs`` and ``synsPerConn`` > 1, weights and delays can still be specified as a single value for all synapses, as a list of length N (each value corresponding to ``synMech`` list index), or a 2D list with outer dimension N (corresponding to ``synMechs``) and inner dimension corresponding to ``synsPerConn``.
+
+	.. image:: figs/multisyn_1.png
+		:width: 35%
+		:align: right
+
+	.. code-block:: python
+
+		netParams.connParams['PYR->PYR'] = {
+			'preConds': {'pop': 'src'}, 'postConds': {'pop': 'dst'},
+			'synMech': ['AMPA', 'GABA'],
+			'sec': 'dend',
+			'synsPerConn': [2, 1],
+			'loc': [[0.5, 0.75], [0.3]],
+			'distributeSynsUniformly': False,
+		}
+
+	Note that ``synsPerConn`` itself can be a list, so that each ``synMech`` can correspond to a distinct number of synapses.
+
+* Usage with **'connList'**
+	If, in addition, you are using **'connList'**-based connectivity (explicit list of connections between individual pre- and post-synaptic cells), then the weights, delays, locs, secs are lists that can be described by up to 3-dimensional lists. The outer dimension now corresponds to the length of ``connList`` (i.e., number of cell-to-cell connections). Then the same logic as above applies to each element of this outer list.
```

Comment:
"If, in addition, you want to use **'connList'**-based connectivity (explicit list of connections between individual pre- and post-synaptic cells), then weights, delays, locs, and secs can be described as lists of up to three dimensions. The outermost dimension will now correspond to the length of ``connList`` (i.e., the number of cell-to-cell connections). The same logic described above will apply to each element of this outer list."